package/yast2.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Mar 28 10:19:44 WEST 2017 - knut.anderssen@suse.com
+
+- backported
+  - SlideShow: Escape plain text release notes being shown properly
+    in RichText (bsc#1028721).
+- 3.1.215
```

Comment:
@teclator no, CASP changes should not go now to SP2. otherwise it will part of next MU, which is what we really do not want.